recipe/patch_yaml/snappy.yaml

```diff
@@ -6,3 +6,13 @@ then:
   - replace_depends:
       old: snappy >=1.1.7,<1.1.8.0a0
       new: snappy >=1.1.7,<2.0.0.0a0
+---
+# For context, see: https://github.com/conda-forge/snappy-feedstock/issues/35
+if:
+  name: "*"
+  has_depends: snappy
+  timestamp_lt: 1712498454329
```

Comment:
For reference, this is the timestamp from `linux-64_snappy-1.2.0-hdb0a2a9_0`'s `index.json`:

```json
{
  "arch": "x86_64",
  "build": "hdb0a2a9_0",
  "build_number": 0,
  "depends": [
    "libgcc-ng >=12",
    "libstdcxx-ng >=12"
  ],
  "license": "BSD-3-Clause",
  "license_family": "BSD",
  "name": "snappy",
  "platform": "linux",
  "subdir": "linux-64",
  "timestamp": 1712498454329,
  "version": "1.2.0"
}
```